### ISSUE TITLE
Update GitDone for new GitHub UI 

### ISSUE THREAD

**Issue by darrenmothersele** [OpenMatt/GitDone#18]
Latest news is that GitHub repos are getting a new look.
https://github.com/blog/2085-a-new-look-for-repositories

At MozFest we spoke about the fragility of coding against unpublished details of the structure of GitHub's markup. 

I'm not sure yet how much the new GitHub repo design will affect the UI rewriting we are doing.

Perhaps some kind of visual regression testing bot to catch when things break?

It looks like it could be possible with Travis CI 
https://travis-ci.org/ 
https://twolfson.com/2014-02-25-visual-regression-testing-in-travis-ci

**Comment by OpenMatt** [comment]
@darrenmothersele Hi, Darren. The new interface is actually much improved -- and actually makes our job much easier, in terms of simplifying the interface for Task Masters. 

## New proposed GitDone interface

Here's a mock-up of what we'll want to do now:

#### 1) Issues page

![gitdone -- new interface 001](https://cloud.githubusercontent.com/assets/1007376/11326138/ffbdd372-912e-11e5-9068-04fbb369bd60.jpg)

and... that's it! This eliminates a lot of clutter and confusion for people who never touch code. 

#### 2) Code page

![code page 001](https://cloud.githubusercontent.com/assets/1007376/11342153/dc8f97b4-91d3-11e5-834f-11df50bb5d2a.jpg)

That's it!

**Comment by OpenMatt** [comment]
Question: is it possible to use the extension to simply have pages like:
https://github.com/OpenMatt/GitDone

re-direct to https://github.com/OpenMatt/GitDone/issues 

instead?

**Comment by darrenmothersele** [comment]
Yes. Let's do that. Is there still to be a way to access the README (in v0.2) without having to disable the addon to get to the repo home page?

**Comment by OpenMatt** [comment]
@darrenmothersele good point.... the README is valuable for some users. So let's set aside the idea of re-directing. And just remove confusing elements from the page, instead. Like this:

![code page 001](https://cloud.githubusercontent.com/assets/1007376/11342153/dc8f97b4-91d3-11e5-834f-11df50bb5d2a.jpg)

**Comment by OpenMatt** [comment]
I cleaned up this ticket to make it clearer. Implementing the two mock-ups above are enough to start testing.

**Comment by darrenmothersele** [comment]
@OpenMatt - I added the code to make these changes. I've left in most of the previous code, I noticed the old UI was still in effect in places when I was logged out of GitHub. I also added an edit button for the project readme on the repo home page, and simplified the readme edit screen a bit. 

I've removed `dist` from the gitignore file so builds are now commited to the repo. You can get the latest build here: https://github.com/OpenMatt/GitDone/raw/master/dist/gitdone.xpi

**Comment by OpenMatt** [comment]
@darrenmothersele amazing! thanks I'll try it out. :)
